integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -9438,8 +9452,8 @@ msgstr "Dieser XLIFF Import ist nicht länger verfügbar."
 #: cms/views/pois/poi_actions.py
 msgid "This location cannot be archived because it is referenced by an event."
 msgstr ""
-"Dieser Ort kann nicht archiviert werden, da er von einer "
-"Veranstaltung verwendet wurde."
+"Dieser Ort kann nicht archiviert werden, da er von einer Veranstaltung "
+"verwendet wurde."
```

Comment:
```suggestion
"verwendet wird."
```
Am I getting this right, that this is supposed to be in past tense not in present tense? :slightly_smiling_face: :thinking: